spec/requests/geoblocking_middleware_spec.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
+# rubocop:disable RSpec/NamedSubject
```

Comment:
disabling this for now as it's outside the scope of this PR (recently added rule), will resolve in a separate PR

Reply:
>_> decided to fix it all here since it's not ballooning the number of changes made.